libs/market-list/src/lib/utils/market-utils.ts

```diff
@@ -5,17 +5,20 @@ import orderBy from 'lodash/orderBy';
 import type { Market, Candle } from '../';
 
 export const totalFees = (fees: Market['fees']['factors']) => {
-  if (!fees) {
-    return undefined;
-  }
-  return formatNumberPercentage(
+  return (
+    fees &&
```

Comment:
non-blocking, question: this change return value form `undefined` to `false` if fees will be falsy. Is this intentional?

Reply:
absolutely not, the intention was to return BigNumber or undefined but I am updating the conditions.

Reply:
updated in last commit.